pkg/planner/core/issuetest/planner_issue_test.go

```diff
@@ -177,3 +182,53 @@ func TestIssue58476(t *testing.T) {
 			`      ├─TableRangeScan(Build) 3333.33 cop[tikv] table:t3 range:(0,+inf], keep order:false, stats:pseudo`,
 			`      └─TableRowIDScan(Probe) 9990.00 cop[tikv] table:t3 keep order:false, stats:pseudo`))
 }
+
+func loadTableStats(fileName string, dom *domain.Domain) error {
+	statsPath := filepath.Join("testdata", fileName)
+	bytes, err := os.ReadFile(statsPath)
+	if err != nil {
+		return err
+	}
+	statsTbl := &util.JSONTable{}
+	err = json.Unmarshal(bytes, statsTbl)
+	if err != nil {
+		return err
+	}
+	statsHandle := dom.StatsHandle()
+	err = statsHandle.LoadStatsFromJSON(context.Background(), dom.InfoSchema(), statsTbl, 0)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func TestIssue50080(t *testing.T) {
```

Comment:
Please don't (1) put the user schema in the public test case and (2) put such a large stats file with the code if unnecessary.
Possible suggestion: 
1. change the column names and put this case into `tests/integrationtest/` where the stats files are compressed.
2. put this case into the private repo.